frontend/src/metabase/common/components/MoveQuestionsIntoDashboardsModal/MoveQuestionsIntoDashboardsModal.tsx

```diff
@@ -44,6 +45,23 @@ export const MoveQuestionsIntoDashboardsModal = withRouter(
     const [bulkMove, bulkMoveReq] =
       useMoveCollectionDashboardQuestionCandidatesMutation();
 
+    // Track the click event when the modal is displayed for the first time
+    useEffect(() => {
+      if (
+        !isAckedInfoLoading &&
+        !ackedInfoStep &&
+        candidatesReq.data?.total !== undefined
```

Comment:
1. It takes a second to grok the intent behind these checks. It would be nice to have them as a named boolean variable that you set outside of the useEffect. Having the boolean be the only dependency also reduces the chance that this over fires.
2. !nit You may consider making `candidatesReq.data?.total !== undefined` => `(candidatesReq.data?.total ?? 0) > 0` to be extra robust / ensure we don't log if there's no candidates. Unlikely to matter, just something I would do.